src/isolate.rs

```diff
@@ -646,6 +656,32 @@ impl Isolate {
     self.set_data_internal(Self::ANNEX_SLOT, annex_ptr as *mut _);
   }
 
+  unsafe fn dispose_annex(&mut self) {
```

Comment:
This moved up to be near the other annex methods, and removing stacks was removed as shared and owned isolates deal with stacks differently.